specification/app/resource-manager/Microsoft.App/preview/2022-06-01-preview/ManagedEnvironments.json

```diff
@@ -963,6 +1006,47 @@
           "description": "Application-specific metadata in the form of key-value pairs."
         }
       }
+    },
+    "workloadProfileStatesCollection": {
+      "description": "Collection of all the workload Profile States for a Premium subscription.",
+      "type": "array",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
```

Comment:
workload profile states is not a tracked resource. It is a proxyOnly resource

Reply:
<img width="261" alt="image" src="https://user-images.githubusercontent.com/41846928/188929393-40242e91-c650-4cea-9ce3-d73f99d4a5c1.png">
Changed to `proxyResource`, is that correct ?